app/Controllers/Http/EventsController.ts

```diff
@@ -1,32 +1,54 @@
 import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
 import Event from 'App/Models/Event'
-import { NO_CONTENT, OK } from 'http-status'
 
 export default class EventsController {
+  // until we don't have a validator, we should keep our requests save this way
+  private eventFields = [
+    'id',
+    'title',
+    'description',
+    'cover',
+    'pre_payment',
+    'box_office',
+    'venue',
+    'links',
+    'creator',
+    'is_public',
+  ]
+
+  // get all events
   public async index({ response }: HttpContextContract) {
     const events = await Event.query()
-    return response.status(OK).json(events)
+    response.json({ success: true, events })
   }
 
+  // get one event per id
   public async show({ request, response }: HttpContextContract) {
     const event = await Event.findByOrFail('id', request.param('id'))
-    return response.status(OK).json(event)
+    response.json({ success: true, event })
   }
 
+  // post one event
   public async store({ request, response }: HttpContextContract) {
-    const event = await Event.create(request.body())
-    return response.json(event)
+    const body = request.only(this.eventFields)
+    const event = await Event.create(body)
+    response.json({ success: true, event })
   }
 
+  // update one event with id
   public async update({ request, response }: HttpContextContract) {
     const event = await Event.findByOrFail('id', request.param('id'))
-    await event.merge(request.body()).save()
-    return response.status(OK).json(event)
+    const body = request.only(this.eventFields)
+
+    await event.merge(body).save()
+    response.json({ success: true, event })
   }
 
+  // delete one event with id
   public async destroy({ request, response }: HttpContextContract) {
     const event = await Event.findByOrFail('id', request.param('id'))
-    await event.delete()
-    return response.status(NO_CONTENT).finish()
+    await event?.delete()
+
+    response.json({ success: true })
```

Comment:
findest du es besser wenn man nicht den status code angibt?

Reply:
Der Statuscode ist trotzdem da. das kannst du in den Tests sehen :)
Den gibt adonis automatisch mit